tests/Aspire.Hosting.Azure.Tests/AzureBicepResourceTests.cs

```diff
@@ -642,36 +613,29 @@ public async Task AddApplicationInsightsWithExplicitLawArgumentDoesntGetDefaultP
         Assert.Equal(expectedManifest, appInsightsManifest.ManifestNode.ToString());
 
         var expectedBicep = """
-            targetScope = 'resourceGroup'
-
-            @description('')
+            @description('The location for the resource(s) to be deployed.')
             param location string = resourceGroup().location
 
-            @description('')
             param applicationType string = 'web'
 
-            @description('')
             param kind string = 'web'
 
-            @description('')
             param logAnalyticsWorkspaceId string
 
-
-            resource applicationInsightsComponent_eYAu4rv7j 'Microsoft.Insights/components@2020-02-02' = {
-              name: toLower(take('appInsights${uniqueString(resourceGroup().id)}', 24))
-              location: location
-              tags: {
-                'aspire-resource-name': 'appInsights'
-              }
-              kind: kind
-              properties: {
-                Application_Type: applicationType
-                WorkspaceResourceId: logAnalyticsWorkspaceId
-              }
+            resource appInsights 'Microsoft.Insights/components@2020-02-02' = {
+                name: take('appInsights-${uniqueString(resourceGroup().id)}', 260)
```

Comment:
triggered, 2 tabs vs 4. What's the idiomatic bicep way?

Reply:
https://learn.microsoft.com/en-us/azure/azure-resource-manager/bicep/file uses 2 spaces.

Reply:
I'd guess https://github.com/Azure/bicep/blob/main/src/Bicep.Core/PrettyPrintV2/PrettyPrinterV2Options.cs#L9 is as definitive as we'll get for an answer on this.  I'd argue that formatter should be ignored because they allow tabs and therefore can't be trusted, but it's a cheap change in Azure.Provisioning.* so I'll update it later.

Reply:
https://github.com/Azure/azure-sdk-for-net/pull/46016 will address this.